### ISSUE TITLE
Mopidy-YouTube Autoplayer

### ISSUE THREAD

**Issue by natumbri** [natumbri/mopidy-youtube#141]
Thanks to @Mohawi, Mopidy-YouTube is getting an 'autoplay' function.  

The initial pull request is in #139, and further changes are in [a new branch](/natumbri/mopidy-youtube/tree/feature/autoplay).

This issue is for beta testing, comments and suggestions for improvements/bug fixes.

**Comment by natumbri** [comment]
@Mohawi - have you had a go at the autoplayer?  I've now moved everything over to the develop branch.  I think it is working quite well.  I have made a couple of changes:
- it works with and without the API key
- it now adds a track when the last track starts playing, rather than when it ends playing, to try to keep the playback continuous 
- it keeps track of what has already been autoplayed, and tries not to repeat previously autoplayed tracks
- there is a config setting to control how far removed from the original track autoplayed tracks get  
- there is a config setting to control the maximum length of autoplayed tracks
